src/config/alwaysOnLibraries.js

```diff
@@ -0,0 +1,21 @@
+import fs from 'fs';
+import path from 'path';
+
+const alwaysOnLibraries = {
```

Comment:
Another possible approach: put SweetAlert in the normal libraries list, but put a flag in the library definition for `alwaysOn` (or maybe `internal`) that suppresses it from the library picker and signals to the preview generator that it should always be included. WDYT?